src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/CodeActions/Models/CodeActionExtensions.cs

```diff
@@ -76,7 +76,7 @@ internal static class CodeActionExtensions
 
         if (!isOnAllowList)
         {
-            razorCodeAction.Title = "(Exp) " + razorCodeAction.Title;
+            razorCodeAction.Title = $"(Exp) {razorCodeAction.Title} ({razorCodeAction.Name})";
```

Comment:
This is unrelated, but when experimenting with code actions, its handy to know what Roslyn calls the action so we know what to add to the allow list.